san/sanbase_graphql_helper.py

```diff
@@ -271,11 +271,17 @@ def _format_from_date(datetime_obj_or_str):
 
 
 def _format_to_date(datetime_obj_or_str):
-    if isinstance(datetime_obj_or_str, datetime.datetime):
-        datetime_obj_or_str = datetime_obj_or_str.isoformat()
+    date_format = '%Y-%m-%d'
+    if not isinstance(datetime_obj_or_str, datetime.datetime):            
```

Comment:
instead of `if not` just use `if` and swap the bodies of the if and else

Reply:
Actually, it can be better handled with an early return:
```python
def _format_to_date(datetime_obj_or_str):
  if isinstance(datetime_obj_or_str, datetime.datetime):
    return datetime_obj_or_str.isoformat()

  try:
    # Throw if the string is not date-formated, parse as date otherwise
    datetime.datetime.strptime(datetime_obj_or_str, date_format)
    dt = iso8601.parse_date(datetime_obj_or_str) + datetime.timedelta(hours=23, minutes=59, seconds=59)
  except:
      dt = iso8601.parse_date(datetime_obj_or_str)

  return dt.isoformat()
```

otherwise you're nesting `try/except` in an `if not` case which is hard to read

Reply:
Thanks, that's better really